thirdparty/cnpy/README.md

```diff
@@ -16,7 +16,7 @@ Loading data written in numpy formats into C++ is equally simple, but requires y
 Default installation directory is /usr/local. 
 To specify a different directory, add `-DCMAKE_INSTALL_PREFIX=/path/to/install/dir` to the cmake invocation in step 4.
 
-1. get [cmake](www.cmake.org)
+1. get [cmake](https://cmake.org/)
```

Comment:
I suppose we need to contribute it to https://github.com/rogersce/cnpy

Reply:
What do you propose?

Reply:
send this change to that repo as well.

Reply:
I contributed this fix for https://github.com/rogersce/cnpy

Reply:
I guess we can resolve this conversation :)